src/platform.ts

```diff
@@ -83,6 +83,8 @@ export class PetSafeSmartFeedPlatform implements DynamicPlatformPlugin {
         homebridgeAccessory =
           this.homebridgeAccessories[uuid] || createHomebridgeAccessory()
 
+      homebridgeAccessory.context = this.config
```

Comment:
I would prefer to pass the config into the accessory constructor, rather than using context. I’m pretty sure context gets serialized and stored when homebridge shuts down, which isn’t necessary in this case

Reply:
Context does get serialized. I can certainly change it to pass in the constructor. I’ll probably work on that tomorrow at this point, as it’s getting a bit late here.